src/components/widgets/shared/BasicAxes.tsx

```diff
@@ -15,11 +15,15 @@ export interface IBasicAxisProps {
 export const BasicHorizontalAxis = (props: IBasicAxisProps) => (
   <ReactVis.XAxis
     tickFormat={props.fnTickFormat}
-    title={props.axisLabel} />
+    title={props.axisLabel}
+    />
+
 );
 
 export const BasicVerticalAxis = (props: IBasicAxisProps) => (
   <ReactVis.YAxis
     tickFormat={props.fnTickFormat}
-    title={props.axisLabel} />
-);
\ No newline at end of file
+    title={props.axisLabel}
+    orientation={props.orientation}
+    />
```

Comment:
a bit nitpicky, but I try to keep my end tags on the same line for consistencies sake